magento/magento2
# Fixed password fields visibility issue for Guest visitor e-mail

### Description (*)

Fixed password field visibility issue for Guest visitor e-mail address
### Fixed Issues (if relevant)

magento/magento2#26903: Guest visitor e-mail in checkout kicks off password field visibility

### Manual testing scenarios (*)

1. add items to cart and reach checkout, use an unregistered e-mail in the e-mail field and fill address data
2. go back to cart page (or to any other page as long outside from checkout page)
3. then go back to the checkout page and see that it presents password fields for authenticating for e-mail that is not registered

### Contribution checklist (*)
